08_book_titles/book.rb

```diff
@@ -0,0 +1,24 @@
+$non_capital_words = %w[a the an] + %w[for and nor but or yet] + %w[above across against along among around at before behind 
```

Comment:
Variables declared using the `$` are global and it is not a good idea to use them. We should better use constants